js/bootstrap-datepicker.js

```diff
@@ -855,7 +855,10 @@
 				html += '<th class="cw">&#160;</th>';
 			}
 			while (dowCnt < this.o.weekStart + 7){
-				html += '<th class="dow">'+dates[this.o.language].daysMin[(dowCnt++)%7]+'</th>';
+				html += '<th class="dow"';
```

Comment:
The class property is closed in the above line, so it will generate incorrent html `<th class="dow" disabled">Mo</th>`